src/Components/AuthBackground/AuthBackground.tsx

```diff
@@ -0,0 +1,27 @@
+import { useLocation } from "react-router-dom";
+import "./AuthBackground.module.scss";
+
+const AuthBackground = () => {
+  const { pathname } = useLocation();
+  return (
+    <>
+      <div
+        className={`auth ${
+          pathname === "/login"
```

Comment:
We can make this chunk of code outside of the JSX, as follows:

```
const pathnameImageClass =        pathname === "/login"
            ? "auth-background"
            : pathname === "/register"
            ? "auth-background2"
            : pathname === "/forgetPassword"
            ? "auth-background3"
            : pathname === "resetPassword"
            ? "auth-background3"
            : "auth-background"
```
   
and the  JSX would be simpler and easier to read ```
<div className={`auth ${pathnameImageClass}`}>
```